src/core/api/index.js

```diff
@@ -26,14 +32,22 @@ function callApi(endpoint, schema, params) {
     .then((response) => normalize(response, schema));
 }
 
-export function search({ page, query }) {
+export function search({ page, query, state }) {
```

Comment:
On reflection I'm not sure about passing the whole state around here. Ditto for `callApi`